src/controllers/PendingTransactionController.ts

```diff
@@ -0,0 +1,50 @@
+
+import { Request, Response } from "express";
+import { sendJSONresponse } from "../common/Utils";
+import { Config } from "../common/Config";
+let Promise = require("bluebird");
+
+export class PendingTransactionController {
+
+    public getPendingTransactions = (req: Request, res: Response) => {
+        const validationErrors: any = this.validateQueryParameters(req);
+        
+        if (validationErrors) {
+            sendJSONresponse(res, 400, {validationErrors});
+            return;
+        }
+
+        const address: string = req.query.address;
+
+        Config.web3.eth.getBlock("pending", true).then((pendings: any) => {
+            return new Promise.filter(pendings.transactions, function(transaction: any) {
+                return (transaction.from === address || transaction.to == address) ? true : false;
```

Comment:
would it be possible to use `TransactionParser`? we can track if it's token transfer then